ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -262,6 +262,13 @@ open class BadgeView: UIView {
         let labelWidth = max(minLabelWidth, min(maxLabelWidth, fittingLabelWidth))
         label.frame = CGRect(x: size.horizontalPadding, y: size.verticalPadding, width: labelWidth, height: labelHeight)
     }
+    
+    open override var backgroundColor: UIColor? {
+        didSet {
+            super.backgroundColor = .clear
```

Comment:
this doesn't seems right to me. if user wants to set the background color custom, it should show that in the background color right? I don't see the difference in your screenshots in the pr description? also, when user change the text or selection state/active nonactive, wouldn't it fall back to the default style color?

Reply:
The PR doesn't allow custom color. My understanding was that we can only set to default presets color based on style. I will update the PR to keep the background color of badge in sync with the base view background color. 

The difference in the snapshot is extra color around the corner edges. This happens because the background color changes the color of the underlying base view color for badge. 

![Screen Shot 2020-10-30 at 12 35 56 PM](https://user-images.githubusercontent.com/63682282/97749604-90256900-1aac-11eb-8761-c0b4d46e173c.png)

I haven't tested the behavior with active/nonactive state but based on my understanding of the [code](https://github.com/microsoft/fluentui-apple/blob/37b39d0f8cb9fa6f71c90ad30d16576a3430c7b4/ios/FluentUI/Badge%20Field/BadgeView.swift#L301 ) the `updateBackground` method handles the underlying state while updating the setting the background color. 

Are there any specific test case you have in mind, that needs to be tested?

Reply:
the demo app should have a case where it shows that client code can change colors of the badgeview